lib/config/router/movie_router.dart

```diff
@@ -0,0 +1,4 @@
+class MovieRouter {
+  static const String movieList = 'MovieList';
+  static const String movieDetails = "MovieDetails";
```

Comment:
Define the routes like` /movie_list` and `/movie_details`

Reply:
And always you need to define an initial route that is `'/'`, for this particular case could be`'/movie_list'` but remember to change it when you add the splash screen.
Replace `'/movie_list'` to `'/'`

Reply:
https://docs.flutter.dev/cookbook/navigation/named-routes

Reply:
Done!